builds/android/app/src/main/res/values-w820dp/dimens.xml

```diff
@@ -0,0 +1,6 @@
+<resources>
+    <!-- Example customization of dimensions originally defined in res/values/dimens.xml
+         (such as screen margins) for screens with more than 820dp of available width. This
+         would include 7" and 10" devices in landscape (~960dp and ~1280dp respectively). -->
+    <dimen name="activity_horizontal_margin">64dp</dimen>
```

Comment:
Hmm this is an auto-generated file by Android Studio, if you change it, does it solve the problem?
Because this problem doesn't occur when you rotate your device, I tend to think that the line 164 in GameBrowserActivityAPI15.java is the source of the problem:
`int nbGamePerRow = configuration.screenWidthDp / THUMBNAIL_HORIZONTAL_SIZE_DPI;`